include/Util/Base64.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef UTIL_BASE64_HPP
+#define UTIL_BASE64_HPP
+
+/*
+ * The code to decode the base64 in compilation time is powered by https://stackoverflow.com/a/59604914
+ * It's used to decode the base64 string in compilation time and improve the efficiency.
+ * Actually it's dirty, but we just treat as a utility and won't maintain it, so just make sure it can work.
+ */
+#include <string_view>
+
+static constexpr size_t DecodeBase64Length(const char *s)
+{
```

Comment:
Sussy indent the left brace should be at EOL.

Reply:
Fix on 2adf576